### PR TITLE
Added Pagination

### PR DESCRIPTION
**Pull request by Rohanmond**

### CODE REVIEW

File: fashify/src/Hooks/FilterHook.js [hunk 1]
```diff
@@ -12,14 +13,23 @@ export const useFilterHook = () => {
...
+
+    newData = priceRangeFilter(newData, priceRange);
+    newData = categoryFilter(newData, categories);
+    newData = sizeFilter(newData, sizes);
+    newData = ratingFilter(newData, rating);
+    newData = sortByPrice(newData, sortBy);
+    return newData;
+  };
+  // const newData = useMemo(
+  //   () => applyFilters(products, filters),
```
Comment (varnita619) [inline fashify/src/Hooks/FilterHook.js]: remove comments

File: fashify/src/Hooks/FilterHook.js [hunk 2]
```diff
@@ -12,14 +13,23 @@ export const useFilterHook = () => {
...
+  //   [filters, products]
+  // );
```
Comment (romabulani) [inline fashify/src/Hooks/FilterHook.js]: Remove these comments

**Review by varnita619 — APPROVED** [review]
Code is modular and clean
With good naming
good to see that you are using useMemo

File: fashify/src/pages/ProductList/components/ProductListMain/ProductListMain.css [hunk 3]
```diff
@@ -19,11 +19,32 @@
...
+.pagination-container {
+  display: flex;
+  margin: 1rem;
+  margin-bottom: 2rem;
+  justify-content: center;
+}
+.pagination-item {
+  cursor: pointer;
+  padding: 0.25rem 1rem;
+  /* font-size: 1.15rem; */
```
Comment (romabulani) [inline fashify/src/pages/ProductList/components/ProductListMain/ProductListMain.css]: This comment also

File: fashify/src/pages/ProductList/components/ProductListMain/ProductListMain.jsx [hunk 4]
```diff
@@ -1,21 +1,79 @@
...
+            .getComputedStyle(gridRef.current)
+            .getPropertyValue('grid-template-columns')
+            .split(' ').length
+      ) *
+      window
+        .getComputedStyle(gridRef.current)
+        .getPropertyValue('grid-template-columns')
+        .split(' ').length
+    );
+  };
```
Comment (romabulani) [inline fashify/src/pages/ProductList/components/ProductListMain/ProductListMain.jsx:30]: Liked that you are dynamically changing the num of products per page

**Review by romabulani — APPROVED** [review]
- Modular code, like that you are using hook for pagination
- Functionality is working as expected
- Good to merge

**Pull request merged**